Add launch handler article (#6908)
GoogleChrome/web.dev: src/site/content/en/blog/declarative-link-capturing/index.md

{% Aside 'caution' %} Declarative Link Capturing was part of the
[capabilities project](/fugu-status/).
The engineering team has decided that Declarative Link Capturing will _not_

> launch in the current shape. Instead, the feature has been redesigned as

Reviewer: ```suggestion
launch with its current design. Instead, the feature has been redesigned as
```

Reviewer: @tomayac Apologies. I didn't release this was a PTAL. I started reviewing as though it were new and found this. We should probably fix it.

link click that is captured? When you think about it, there are many ways links and navigations can
be captured, including, but not limited to, the following:

- Clicked links from other web pages.
- URL launches from a platform-specific app in the operating system.

> - Navigations originating from the [App Shortcuts API](/app-shortcuts/)
> - Links that go through [URL protocol handlers](/url-protocol-handler/)
> - Navigations caused by [file handlers](/file-handling/)
> - Navigations caused by the [Share Target API](/web-share-target/)
> - …and others

Reviewer: ```suggestion
- Navigations originating from the [App Shortcuts API](/app-shortcuts/).
- Links that go through [URL protocol handlers](/url-protocol-handler/).
- Navigations caused by [file handlers](/file-handling/).
- Navigations caused by the [Share Target API](/web-share-target/).
- …and others.
```
